proto/Makefile

```diff
@@ -16,35 +16,22 @@
 # Author: Spencer Kimball (spencer.kimball@gmail.com)
 # Author: Andrew Bonventre (andybons@gmail.com)
 
-PROTO_LIB   := lib/libroachproto.a
-PROTOS      := api.proto config.proto data.proto errors.proto gossip.proto heartbeat.proto internal.proto
+PROTOC      := $(shell git rev-parse --show-cdup)_vendor/usr/bin/protoc
```

Comment:
Ah, didn't realize we had other Makefiles. Why aren't the files generated by those Makefiles being checked in?  All the proto makefiles are full of `../../..` paths (with varying numbers of directories); we should either replace all of those with the repo root extracted from git or just keep using `..` for everything. I have a slight preference for `..`; embedding version-control commands in the build feels wrong.